### PR TITLE
Add ocVerifyPod and ocGetLogsFromPods

### PR DESCRIPTION
**Pull request by p3ck**
These methods assume you already called openshift.withcluster() and
openshift.withproject().  These should be used when doing parallel
jobs.

### CODE REVIEW

**Comment by p3ck** [comment]
[merge]

File: src/org/centos/pipeline/PipelineUtils.groovy [hunk 1]
```diff
@@ -756,6 +756,46 @@ def executeInContainer(String stageName,
...
 }
 
+/**
+ *
+ * @param nodeName podName we are going to verify.
+ * @return
+ */
+def ocVerifyPod(String nodeName) {
+    def describeStr = openshift.selector("pods", nodeName).describe()
+    out = describeStr.out.trim()
```
Comment (robnester-rh) [inline src/org/centos/pipeline/PipelineUtils.groovy]: I _think_ you need a ```def``` here?

File: src/org/centos/pipeline/PipelineUtils.groovy [hunk 2]
```diff
@@ -756,6 +756,46 @@ def executeInContainer(String stageName,
...
+
+    sh 'mkdir -p podInfo'
+
+    writeFile file: 'podInfo/node-pod-description-' + nodeName + '.txt',
```
Comment (robnester-rh) [inline src/org/centos/pipeline/PipelineUtils.groovy]: nit: it's more groovy-esque to do ```"podInfo/node-pod-description-${nodeName}.txt"```

File: src/org/centos/pipeline/PipelineUtils.groovy [hunk 3]
```diff
@@ -756,6 +756,46 @@ def executeInContainer(String stageName,
...
+                text: out
+    archiveArtifacts 'podInfo/node-pod-description-' + nodeName + '.txt'
```
Comment (robnester-rh) [inline src/org/centos/pipeline/PipelineUtils.groovy]: nit: as above.

**Pull request merged**